samples/excel/85-preview-apis/data-types-web-image.yaml

```diff
@@ -0,0 +1,206 @@
+order: 2
+id: excel-data-types-web-image
+name: 'Data types: Web images'
+description: This sample shows how to set and get data types using the web image properties.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#insert-image").click(() => tryCatch(insertImage));
+        $("#retrieve-image-info").click(() => tryCatch(retrieveImageInfo));
+        $("#open-image").click(() => tryCatch(openImage));
+        $("#clear-form").click(() => tryCatch(clearForm));
+
+        async function insertImage() {
+          await Excel.run(async (context) => {
+            // This method inserts a web image into the currently selected cell.
+
+            // Retrieve image data from the input fields and then clear the input fields.
+            const imageUrl = $("#url").val() as string;
+            const imageAltText = $("#alt-text").val() as string;
+            clearForm();
+
+            // Load the active cell.
+            const activeCell = context.workbook.getActiveCell();
+            activeCell.load();
+            await context.sync();
+
+            if (!imageUrl) {
+              console.log("Please enter an image URL.");
+              return;
+            }
+
+            // Insert web image into the active cell.
+            activeCell.valuesAsJson = [
```

Comment:
Instead of directly assigning the JSON, consider creating a WebImageCellValue variable as an intermediary step. That way, the type is clear and readers in the ref docs can know what to look up.